pallets/evm-accounts/src/lib.rs

```diff
@@ -376,3 +525,14 @@ where
 		ApprovedContract::<T>::contains_key(evm_address)
 	}
 }
+
+#[cfg(feature = "std")]
+/// Used for testing purposes.
+/// Signs a message for `claim_account` with a given pair of keys.
+pub fn sign_message<T: Config>(pair: sp_core::sr25519::Pair, account: &T::AccountId, asset_id: T::AssetId) -> Signature
+where
+	T::AccountId: AsRef<[u8; 32]> + frame_support::traits::IsType<AccountId32>,
+{
+	let signature = pair.sign(Pallet::<T>::create_claim_account_message(account, asset_id).as_slice());
+	Signature::Sr25519(signature)
+}
```

Comment:
why we need it here when it is only for testing purposes?

Reply:
so it can be used in integration tests.